### PR TITLE
Use official API for wikipedia

### PR DESCRIPTION
**Pull request by samiyaakhtar**

### CODE REVIEW

**Comment by MeenuV** [comment]
@samiyaakhtar can you post some sample cards/json responses?

File: client/scraper/wikipedia_metadata.py [hunk 1]
```diff
@@ -20,7 +24,10 @@ def fetch_site_data(self, sanitized_url, status_code):
         if path is not None:
             title = path.group(1)
             try:
-                page = wikipedia.page(title)
+                web_request = requests.get(self.fetch_data_url + title)
```
Comment (MeenuV) [inline client/scraper/wikipedia_metadata.py]: call this `api_request` instead?

File: client/scraper/wikipedia_metadata.py [hunk 2]
```diff
@@ -31,20 +38,19 @@ def fetch_site_data(self, sanitized_url, status_code):
     def parse_content(self, response):
         self.generic_parse_content(response)
 
-        page = response.wiki_page
+        keys = response.wiki_page.keys()
+        page = response.wiki_page[keys[0]]
 
-        if page is not None and page.summary is not None:
-            self.prop_map[FieldKeyword.DESC] = page.summary
+        if page is not None and "extract" in page:
```
Comment (MeenuV) [inline client/scraper/wikipedia_metadata.py]: Can you make `extract` a constant?

File: client/scraper/wikipedia_metadata.py [hunk 3]
```diff
@@ -1,8 +1,10 @@
+from bs4 import BeautifulSoup
```
Comment (AliceYuan) [inline client/scraper/wikipedia_metadata.py]: can you split the imports like
third party imports

python library imports

in app imports

**Comment by AliceYuan** [comment]
@samiyaakhtar I changed some stuff in your commit (mostly nitpicky stuff about splitting the function up that would have been too hard to explain in comments) the CR have not been addressed & you should take a look at the code I wrote to see it makes sense

**Comment by samiyaakhtar** [comment]
Thanks @AliceYuan @MeenuV I made some changes you guys suggested and I looked over your changes alice :+1:

File: client/scraper/wikipedia_metadata.py [hunk 4]
```diff
@@ -1,16 +1,35 @@
-import collections
-import re
+from bs4 import BeautifulSoup
 from urlparse import urlparse
```
Comment (AliceYuan) [inline client/scraper/wikipedia_metadata.py]: urlparse is in python lib! (anything that is not in requirements should be in python lib)

**Comment by AliceYuan** [comment]
r+ other than the small thing I added (don't forget to merge commits)

**Pull request merged**